impl/src/main/java/org/ehcache/config/builders/ExpiryPolicyBuilder.java

```diff
@@ -64,36 +65,67 @@
     return new TimeToIdleExpiryPolicy(timeToIdle);
   }
 
+  @FunctionalInterface
+  public interface TriFunction<T, U, V, R> {
+    /**
+     * Applies this function to the given arguments.
+     *
+     * @param t the first function argument
+     * @param u the second function argument
+     * @param v the third function argument
+     * @return the function result
+     */
+    R apply(T t, U u, V v);
+  }
+
   /**
    * Fluent API for creating an {@link ExpiryPolicy} instance where you can specify constant values for creation, access and update time.
    * Unspecified values will be set to {@link ExpiryPolicy#INFINITE INFINITE} for create and {@code null} for access and update, matching
    * the {@link #noExpiration()}  no expiration} expiry.
    *
    * @return an {@link ExpiryPolicy} builder
    */
-  public static ExpiryPolicyBuilder expiry() {
-    return new ExpiryPolicyBuilder();
+  public static <K, V> ExpiryPolicyBuilder<K, V> expiry() {
```

Comment:
I think this is an error in my generics changes... I think this should return `ExpiryPolicyBuilder<Object, Object>`.

Reply:
Why?